### ISSUE TITLE
404 on MathJax/config/Safe.js

### ISSUE THREAD

**Issue by blitzd** [jupyter/notebook#314]
I'm getting a 404 starting up Jupyter with just a 'jupter notebook' command in Anaconda on Windows 10 (though the same thing happens to my colleague on Windows 8).

```
404 GET /static/components/MathJax/config/Safe.js?rev=2.5.3 (::1) 7.03ms referer=http://localhost:8889/notebooks/Testing/Untitled5.ipynb
```
Nothing else loads in the notebook after that, so it's unusable.

I also get the same issues when starting it with 'ipython notebook'

Info:

```
{'commit_hash': 'f534027',
 'commit_source': 'installation',
 'default_encoding': 'cp850',
 'ipython_path': 'C:\\Anaconda3\\envs\\jupyter\\lib\\site-packages\\IPython',
 'ipython_version': '4.0.0',
 'os_name': 'nt',
 'platform': 'Windows-8-6.2.9200',
 'sys_executable': 'C:\\Anaconda3\\envs\\jupyter\\python.exe',
 'sys_platform': 'win32',
 'sys_version': '3.4.3 |Continuum Analytics, Inc.| (default, Mar  6 2015, '
                '12:06:10) [MSC v.1600 64 bit (AMD64)]'}
```

freeze for the env:

```
certifi==14.5.14
decorator==4.0.2
ipykernel==4.0.3
ipython==4.0.0
ipython-genutils==0.1.0
ipywidgets==4.0.2
Jinja2==2.8
jsonschema==2.4.0
jupyter==1.0.0
jupyter-client==4.0.0
jupyter-console==4.0.0
jupyter-core==4.0.4
MarkupSafe==0.23
mistune==0.7
nbconvert==4.0.0
nbformat==4.0.0
notebook==4.0.2
numpy==1.9.2
pandas==0.16.2
path.py==0.0.0
pickleshare==0.5
Pygments==2.0.2
pyreadline==2.0
python-dateutil==2.4.2
pytz==2015.4
pyzmq==14.7.0
qtconsole==4.0.1
scipy==0.16.0
simplegeneric==0.8.1
six==1.9.0
tornado==4.2.1
traitlets==4.0.0
wheel==0.24.0
```

Also can't try master because it's failing to install for me (moving that over to a separate issue).

**Comment by blitzd** [comment]
This is directly related to #315 and build issues, I've informed the Anaconda folks on their google group of the issue.

**Comment by minrk** [comment]
closed by #316, released as 4.0.3

**Comment by minrk** [comment]
I did something a bit wrong, I'll cut 4.0.4 shortly with the right fix.
